extensions/interactions/EndExploration/directives/end-exploration-rules.service.spec.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for end exploration rules service.
+ */
+
+import { EndExplorationRulesService } from
+// eslint-disable-next-line max-len
+  'interactions/EndExploration/directives/end-exploration-rules.service';
+
+describe('End exploration rules service', () => {
+  let endExplorationRulesService: EndExplorationRulesService = null;
+
+  beforeEach(() => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    endExplorationRulesService = new EndExplorationRulesService();
```

Comment:
This feels very weird... How does this cover the file?

Reply:
@nithusha21 the file contain only class, not its definition so I try to test it just creating an object, so its there any other way?

Reply:
Hmmm, that is weird. Firstly, if it does cover it, then why haven't you removed it from check_frontend_coverage.py (there is a list of files which are not yet at 100% coverage, and you need to remove from the list. Also, if it isn't in the list, then this test is unnecessary).